supervisord.conf

```diff
@@ -35,7 +35,7 @@ autorestart=false
 command=cron -f
 
 [program:research-app]
-command=pserve development.ini
+command=./manage.py devserver
```

Comment:
This will lead to "dev server running in prod" as deployed through docker. I'll merge here but add an issue to correct this.